app/workers/standup_meeting_group/determine_missing_standup_meetings_worker.rb

```diff
@@ -0,0 +1,45 @@
+class StandupMeetingGroup
+  class DetermineMissingStandupMeetingsWorker < ApplicationWorker
+    sidekiq_options queue: 'default'
+
+    def perform(start_minutes = 0, end_minutes = 0)
+      start_time = start_minutes.minutes.ago
+      end_time = end_minutes.minutes.from_now
+      current_date = Date.current
+
+      # NOTE: the goal is to compare the number of users in a standup_meeting_group to the number of standup_meetings
+      # for a particular day. If there are more users than standup_meetings, it means people haven't completed it yet.
+      # Thus, we need to create standup_meetings (drafts) for those users and send a notification.
+      standup_meeting_groups =
+        StandupMeetingGroup.left_outer_joins(:standup_meetings)
+                           .joins(:standup_meeting_groups_users)
+                           .preload(:standup_meetings)
+                           .select(
+                            'standup_meeting_groups.id,
+                            ARRAY_AGG(standup_meeting_groups_users.user_id) AS user_ids'
+                          )
+                           .where(active: true)
+                           .where(
+                             'standup_meeting_groups.start_time::time BETWEEN ?::time AND ?::time',
+                             start_time,
+                             end_time
+                           )
+                           .where('standup_meetings.id IS NULL OR standup_meetings.meeting_date = ?', current_date)
+                           .group('standup_meeting_groups.id')
+                           .having('COUNT(standup_meeting_groups_users.id) > COUNT(DISTINCT standup_meetings.id)')
+
+      # NOTE: later on if this becomes a bottle neck we can have a separate worker
+      # for each standup_meeting_group (and pass in user_ids).
+      # Ideally though we would just batch these at that point.
+      standup_meeting_groups.each do |standup_meeting_group|
```

Comment:
That makes sense to me -- I think we're going to have very few standup meeting groups, overall, so despite the O(N^2), I'd say it's probably good enough. I guess we'll just monitor sidekiq for failing jobs to identify if/when this becomes problematic?